specs/APIReview_TextureStream.md

```diff
@@ -0,0 +1,705 @@
+interface ICoreWebView2StagingTextureStreamStoppedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event. There are no event args and the args
+  /// parameter will be null.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] IUnknown* args);
+}
+/// This is the callback for texture stream rendering error.
+[uuid(52cb8898-c711-401a-8f97-3646831ba72d), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamErrorReceivedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] ICoreWebView2StagingTextureStreamErrorReceivedEventArgs* args);
+}
+/// This is the event args interface for texture stream error callback.
+[uuid(0e1730c1-03df-4ad2-b847-be4d63adf700), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamErrorReceivedEventArgs : IUnknown {
+  /// Error kind.
+  [propget] HRESULT Kind([out, retval]
+      COREWEBVIEW2_TEXTURE_STREAM_ERROR_KIND* value);
+  /// texture that the error is associated with.
+  [propget] Texture([out, retval] ICoreWebView2StagingTexture** value);
+}
+[uuid(431721e0-0f18-4d7b-bd4d-e5b1522bb110), object, pointer_default(unique)]
+interface ICoreWebView2StagingRenderAdapterLUIDChangedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] IUnknown* args);
+}
+[uuid(431721e0-0f18-4d7b-bd4d-e5b1522bb110), object, pointer_default(unique)]
+interface ICoreWebView2StagingRenderAdapterLUIDChangedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingEnvironment * sender,
+      [in] IUnknown* args);
+}
+/// This is the callback for web texture.
+[uuid(9ea4228c-295a-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamWebTextureReceivedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] ICoreWebView2StagingTextureStreamWebTextureReceivedEventArgs* args);
+}
+
+/// This is the event args interface for web texture.
+[uuid(a4c2fa3a-295a-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamWebTextureReceivedEventArgs : IUnknown {
+  /// Return ICoreWebView2StagingWebTexture object.
+  /// The call does not create new ICoreWebView2StagingWebTexture object, instead
+  /// returns the same object.
+
+  /// The texture handle will be reused when ICoreWebView2StagingWebTexture

```

Comment:
This seems like good info for the WebTexture's property documentation rather than this. Please remove it from here and I'll add it to the property docs in WebTexture.